azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/utilities/implementation/FeatureFlags.kt

```diff
@@ -28,6 +28,7 @@ There is 2 parts to this system.
   This is recommended over "values()" which will only access the Enum values.
 
  */
+
 enum class FeatureFlags(
```

Comment:
would it be possible to move the class to demo app?

Reply:
Have moved.